app/models/sign_in/client_config.rb

```diff
@@ -33,6 +33,10 @@ def cookie_auth?
       authentication == Constants::Auth::COOKIE
     end
 
+    def refresh_token_path
+      Constants::Auth::REFRESH_ROUTE_PATH
+    end
```

Comment:
This is overriding the column method. Whenever you call `config.refresh_token_path` it will always call this method which returns the default and not what's in the database. You should avoid overriding column methods as it causes unexpected behavior.